Norm_end_to_end.Rmd

```diff
@@ -199,11 +264,11 @@ pandoc.table(intern_standard[, c("name", "mean_abd", "sd_abd", "CV_raw")],
 Implement between-sample normalization using filled-in features. This process
 effectively mitigates variations influenced by technical issues, such as
 differences in sample preparation and injection methods. In this case we will
-test different method that are commonly used in metabolomics. 
+test different method that are commonly used in metabolomics.
 
-### MRM: Median Ratio Method
```

Comment:
I'm not super sure we should call this MRM - in fact, the process we use here is called *median scaling* in the normalization *world*. The MRM (AFAIK) is a function from the DESeq2 package that works slightly different.

Reply:
do you want me to look into the actual MRM function ?